source/process/security.md

```diff
@@ -88,27 +91,166 @@ The following resources are monitoring for information about new security threat
+   - Yes.
+
+1. Are there written network password policies and/or procedures?
+   - Yes.
+
+1. Is password administration employed for critical systems? 
+   - Yes.
+
+1. Are passwords prevented from being displayed in clear text during user authentication or in electronic/printed reports?
+   - Yes.
 
 1. If user accounts are assigned to non-permanent personnel (e.g., contractors, consultants)  for troubleshooting purposes, are the accounts disabled or removed after each use?
    - Yes
 
+### Operational Security
+
+1. Is there a risk assessment program that has been approved by management, communicated to appropriate constituents and an owner to maintain and review the program?
+   - Yes.
+
+1. Is there an information security policy that has been approved by management, communicated to appropriate constituents and an owner to maintain and review the policy?
+   - Yes.
+
+1. Is there a vendor management program?
+   - Yes.
+
+1. Is there a respondent information security function responsible for security initiatives?
+   - Yes.
+
+1. Is there an asset management policy or program that has been approved by management, communicated to appropriate constituents and an owner to maintain and review the policy?
+   - Yes.
+
+1. Are Management approved operating procedures utilized?
+   - Yes.
+
+1. Is there an operational change management / change control policy or program that has been approved by management, communicated to appropriate constituents and an owner to maintain and review the policy?
+   - Yes.
+
+1. Are system backups performed?
+   - Yes.
+
+1. Are firewalls in use for both internal and external connections?
+   - Yes.
+
+1. Are firewalls or IPS(s) secured against unauthorized access from the Internet, Extranet and Intranet users?
+   - Yes.
+
+1. Are vulnerability assessments, scans or penetration tests performed on internal or external networks?
+   - Yes.
+
+1. Are incoming e-mails scanned for questionable file attachments?
+   - Yes.
+
+1. Does the company use spam filtering software to reduce the amount of unsolicited e-mails?
+   - Yes.
+
+1. Are e-mail attachments scanned by the anti-virus?
+   - Yes.
+
+### Business Resiliency
+
+For more information on Business Resiliency, see the [Mattermost Business Continuity Plan](https://docs.mattermost.com/process/security.html#business-continuity-plan).
+
+1. Is there an established Business Resiliency program that has been approved by management and communicated to appropriate constituents?
+   - Yes.
+
+1. Has a Business Impact Analysis been conducted?
+   - Yes.
+
+1. Is there a formal process focused on identifying and addressing risks of disruptive incidents to the organization?
+   - Yes.
+
+1. Is there an established Business Resiliency program that has been approved by management and communicated to appropriate constituents?
+   - Yes.
+
+1. Are specific response and recovery strategies defined for the prioritized activities?
```

Comment:
What does 'prioritized activities' refer to?

Reply:
Maybe we can re-word it to `Are specific response and recovery strategies defined for addressing risks of disruptive incidents to the organization.`

Reply:
Sounds good, I've updated.